Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># IntelliJ\r\n*.iml\r\n/.idea/caches\r\n/.idea/libraries\r\n/.idea/modules.xml\r\n/.idea/workspace.xml\r\n/.idea/navEditor.xml\r\n/.idea/assetWizardSettings.xml\r\n/.idea/kotlinc.xml\r\n/.idea/gradle.xml\r\n/.idea/compiler.xml\r\n/.idea/misc.xml\r\n\r\n# Gradle files\r\n.gradle/\r\nbuild/\r\nout/\r\n\r\n# Local configuration file (sdk path, etc)\r\nlocal.properties\r\ndevelop.properties\r\n\r\n# Gradle-profiler data\r\nprofile-out*/\r\ngradle-user-home/\r\n*.hprof\r\n\r\n# Android Studio generated files and folders\r\ncaptures/\r\n.externalNativeBuild/\r\n.cxx/\r\noutput.json\r\n\r\n# Built application files\r\n*.apk\r\n*.aar\r\n*.ap_\r\n*.aab\r\n\r\n# Keystore files\r\n*.jks\r\n*.keystore\r\n\r\n# Google Services (e.g. APIs or Firebase)\r\ngoogle-services.json\r\n\r\n# OS specific\r\n.DS_Store\r\n*.log\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision a9a664f1a2368d22e0aa9616f2b2ec8b85da70ac)
+++ b/.gitignore	(date 1731696291980)
@@ -1,4 +1,6 @@
 # IntelliJ
+**/.idea/
+
 *.iml
 /.idea/caches
 /.idea/libraries
